dronekit_sitl/__init__.py

```diff
@@ -137,14 +138,59 @@ def check_complete(count, block_size, total_size):
     if verbose:
         print('Ready to boot.')
 
+sitl_instance_count = 0
+
+class ArdupilotCapabilities():
```

Comment:
Can you please make a base class for this? perhaps `class Capabilities`, and perhaps standardize the known interfaces like `using_sim | model | has_defaults_path` forcing anyone else to use the same